internal/console/pre_scan.go

```diff
@@ -148,12 +148,8 @@ func (console *console) preScan() {
 		log.Info().Msgf("Total memory: %s", bytefmt.ByteSize(mem.Total))
 	}
 
-	cpu, err := consoleHelpers.GetNumCPU()
-	if err != nil {
-		log.Err(err).Msg("failed to get CPU")
-	} else {
-		log.Info().Msgf("CPU: %.1f", cpu)
-	}
+	cpu := consoleHelpers.GetNumCPU()
+	log.Info().Msgf("CPU: %.1f", cpu)
```

Comment:
Should we print the CPU even if it is set to -1?

Reply:
It will never happen that the value is -1. When I return -1 on the function, it also return an error and if there is an error, it tries to get the cpus number in other ways until the last case, returning `float32(runtime.NumCPU())`. This will never be -1